app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -0,0 +1,26 @@
+class DradisDatatable {
+  constructor(selector) {
+    this.$table = $(selector);
+    this.dataTable = null;
+    this.init();
+  }
+
+  init() {
+    var dataTable = this.$table.DataTable({
+      pageLength: 25,
+      lengthChange: false
+    });
+
+    // Assign the instantiated DataTable as a DradisDatatable property
+    this.dataTable = dataTable;
+    this.behaviors();
+  }
+
+  behaviors() {
```

Comment:
the function name is too generic. how about `behaviorOnLeave()` or `unbindDatatable()`?

Reply:
Updated, thanks